DBlock Game/DBlock Game/Program.cs

```diff
@@ -1570,6 +1593,11 @@ public static void TheRiddles()
             TextBox("The vial is labeled V..U..HN  you cant make out the full name..", Player, false);
             TextBox("You put the vial in your pocket", Player, false);
             TextBox("Does this go in anything? A drink perhaps?", Player, false);
+            TextBox("You leave the room", Player, false);
+            // WestHallway_Basement(); to be uncommented once  WestHallway_Basement has been pushed to main
+
```

Comment:
I like the commenting out methods from other branches till it's going to be merged so people can see the intended function going to be added but still easily allows reviewers to run and test code without it not compling